libraries/BotKit Samples/Microsoft.Bot.Sample.Slack/Bots/EchoBot.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I don't think we want this sample in this repo. Can you pull this out of the PR?

Reply:
We removed the sample. Thanks!